### PR TITLE
Add heroku migration draft

### PR DESCRIPTION
**Pull request by danieltprice**
Migrate from Heroku
https://websitemain62807-dpriceherokumigration.gtsb.io/docs/how-to-guides/hasura-heroku-migration/

### CODE REVIEW

File: content/docs/how-to-guides/hasura-heroku-migration.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
+---
+title: Migrate from Hasura with Heroku to Hasura with Neon 
+enableTableOfContents: true
+isDraft: true
+---
+
+This guide describes how to migrate your data from a Hasura with Heroku PostgreSQL to Hasuru with Neon PostgreSQL.
```
Comment (duskpoet) [inline content/docs/how-to-guides/hasura-heroku-migration.md:7]:
```suggestion
This guide describes how to migrate your data from a Hasura with Heroku PostgreSQL to Hasura with Neon PostgreSQL.
```

File: content/docs/how-to-guides/hasura-heroku-migration.md [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+To retrieve your Neon connection string from the Hasura Console:
+
+1. Log into your Hasura account.
+2. Select the project that you want to migrate and click **Launch Console**.
+3. Select the **DATA** tab.
+4. Copy the Neon connection string.
+
+To retrieve your Neon connection string from the Neon Console:
+
+1. Log in to the [Neon Console](https://console.neon.tech/app/projects).
```
Comment (duskpoet) [inline content/docs/how-to-guides/hasura-heroku-migration.md:54]: That method would not work, cause we do not expose the password

**Review by duskpoet — COMMENTED** [review]
Also worth mentioning, that heroku migration part is useful on its own, cause heroku shuts down the free tier for everyone not only just hasura

File: content/docs/how-to-guides/hasura-heroku-migration.md (hunk 2 above)
Comment (raoufchebri) [reply content/docs/how-to-guides/hasura-heroku-migration.md:54]: Instead of select a project, we should create a project a copy the connection string

Comment (danieltprice) [reply content/docs/how-to-guides/hasura-heroku-migration.md]: Changed to creating a project.

**Review by duskpoet — APPROVED** [review]

**Pull request merged**